Automattic/sensei
# Add REST API for adding students to courses

Fixes part of #4959

### Changes proposed in this Pull Request

* Add REST API endpoint to add users to courses
* *Tests for error paths TBD*

### Testing instructions

* Get WP REST nonce (make sure you're logged in): http://devwp.local/wp-admin/admin-ajax.php?action=rest-nonce (replace devwp.local with your domain)
* Copy Cookie name and value
* Execute in terminal (replace WPNONCE, COOKIE-NAME & COOKIE-VALUE, USER_ID, COURSE_ID with actual values):
```
curl -X "POST" "http://devwp.local/?rest_route=%2Fsensei-internal%2Fv1%2Fcourse-users%2Fbatch&_wpnonce=WPNONCE" \
     -H 'Cookie: COOKIE-NAME=COOKIE-VALUE' \
     -H 'Content-Type: application/json; charset=utf-8' \
     -d $'{
  "user_ids": USER_ID,
  "course_ids": COURSE_ID
}'
```
* Make sure you get status code 200 and your user is enrolled for course.
